sdk/purview/purview-datamap-rest/review/purview-datamap.api.md

```diff
@@ -6,10 +6,6 @@
 
 import type { Client } from '@azure-rest/core-client';
 import type { ClientOptions } from '@azure-rest/core-client';
-import { createFile } from '@azure/core-rest-pipeline';
-import { createFileFromStream } from '@azure/core-rest-pipeline';
```

Comment:
@timovv is this expected to removed this for mfd?

Reply:
Yes, this is expected.